spec/system/consumer/authentication_spec.rb

```diff
@@ -108,6 +108,21 @@
             expect(page).to have_content "doesn't match"
           end
 
+          it "Failing to sign up because the user is too quick" do
+            InvisibleCaptcha.timestamp_enabled = true
+            InvisibleCaptcha.timestamp_threshold = 30
+
+            fill_in "Your email", with: user.email
+            fill_in "Choose a password", with: "test12345"
+            fill_in "Confirm password", with: "test12345"
+            click_signup_button
+
+            expect(page).to have_content "Sorry, that was too quick! Please resubmit."
```

Comment:
Does it work to use Timecop to go into the future and submit again?

Reply:
It doesn't due to how InvisibleCaptcha works, it will delete the timestamp in the session before doing the check, so if you try to resubmit Invisible Captcha will complain about a missing timestamp in the logs. There is no feedback to the user though, so it could be a bit confusing, hopefully they have the reflex to reload the page and try again.

Reply:
It makes me a bit worried that we don't test the success state. I don't understand the comments here, but will give it a go myself to see if it can work..